js/src/immersive-reader-sdk.ts

```diff
@@ -5,7 +5,18 @@ import { renderButtons } from './renderButtons';
 import { close, launchAsync } from './launchAsync';
 
 window.addEventListener('load', () => {
+    if (!('Promise' in window)) {
```

Comment:
can we add a comment to explain what this if statement is checking?

Reply:
I considered adding a comment, but on second thought the code is pretty self-documenting if you're familiar with `window`.